### PR TITLE
Support destroy() method

### PR DESCRIPTION
**Pull request by netil**
## Issue

#152 

## Details

Functionality to release resources and events attached

## Preferred reviewers

@sculove @jongmoon

### CODE REVIEW

File: src/flicking.js [hunk 1]
```diff
@@ -1459,6 +1459,26 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
...
+		},
+
+		/**
+		 * Release resources and events attached
+		 * @ko 사용된 리소스와 이벤트를 해제
+		 * @method eg.Flicking#destroy
+		 */
+		destroy: function() {
+			// remove applied inline style and unwrap container element
+			this.$wrapper.attr("style", "");
```
Comment (sculove) [inline src/flicking.js]: How about restore previous style values?

Comment (jongmoon) [inline src/flicking.js]: @sculove I don't want to restore previous style value. I think It would makes messy.

Comment (sculove) [inline src/flicking.js]: if you accept my opinion, you'd better fix it later.

File: test/unit/js/flicking.test.js [hunk 2]
```diff
@@ -813,6 +813,36 @@ test("enaableInput() / disableInput()", function(assert) {
...
+
+	// When
+	inst.destroy();
+
+	simulator(el, {
+		deltaX: -70
+	}, function() {
+		assert.ok(!isEventFired, "Input action should be disabled.");
+		assert.ok($(el).find("."+ containerClassName).length === 0, "Container element was removed?");
+
```
Comment (jongmoon) [inline test/unit/js/flicking.test.js]: How about to check the style initialized?

**Comment by jongmoon** [comment]
LGTM!

**Comment by sculove** [comment]
LGTM

**Pull request closed without merging**